src/pybind/mgr/dashboard/frontend/src/app/ceph/rgw/rgw-overview-dashboard/rgw-overview-dashboard.component.ts

```diff
@@ -60,8 +62,8 @@ export class RgwOverviewDashboardComponent implements OnInit, OnDestroy {
   loading = true;
   multisiteSyncStatus$: Observable<any>;
   subject = new ReplaySubject<any>();
-  syncCardLoading = true;
   fetchDataSub: Subscription;
+  errorMessage: string;
```

Comment:
i only see this being used inside the place where you've defined it. So assigning a const inside that function should be enough isn't it?